.github/workflows/functional-arm64.yaml

```diff
@@ -0,0 +1,30 @@
+name: functional-arm64-tests
+on: [push, pull_request]
+permissions: read-all
+jobs:
+  test:
+    runs-on: [Linux, ARM64]
+    strategy:
+      fail-fast: true
+      matrix:
+        target:
+          - linux-arm64-functional
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # v3.5.0
+        with:
+          go-version: "1.19.5"
+      - run: date
+      - env:
+          TARGET: ${{ matrix.target }}
+        run: |
+          echo "${TARGET}"
+          case "${TARGET}" in
+            linux-arm64-functional)
+              GO_BUILD_FLAGS='-v -mod=readonly' ./scripts/build.sh && GOARCH=arm64 PASSES='functional' ./scripts/test.sh
```

Comment:
We are going to deprecate the functional test, FYI. https://github.com/etcd-io/etcd/issues/15102

If we really need to promote arm64 to tier 1 support, I would suggest to extend the all or majorities of existing workflow to cover `ARM64`. cc @ptabor @serathius @spzala

Reply:
@ahrtr of course! I am requesting @geetasg and @chaochn47 who will be helping out to identify and add what ever jobs needed.

Reply:
Sure, it's OK to do it in a separate PR.